packages/qa/tests/features/studyDetailsTests/studyDetailsSummaryTest.e2e.ts

```diff
@@ -202,7 +202,7 @@ test.describe('Access Study Details Page and view Summary - @se_26', () => {
     })
   })
 
-  test('Estimated reopening date is only Displayed for Suspended Studies - @se_26_estimated', async ({
+  test.skip('Estimated reopening date is only Displayed for Suspended Studies - @se_26_estimated', async ({
```

Comment:
Not an action but don't think we want to pull these skips into se-phase-2 - I'll keep an eye on them and un skip in my next PR